tests/test_db.py

```diff
@@ -11,60 +11,76 @@
 
 class DBOpTests(SimpleTestCase):
     def test_create(self):
-        op = DBOp("myalias", "SELECT 1")
+        op = DBOp("myalias", "SELECT 1", None)
         assert op.alias == "myalias"
         assert op.query == "SELECT 1"
+        assert op.tb is None
 
     def test_equal(self):
-        assert DBOp("foo", "bar") == DBOp("foo", "bar")
+        assert DBOp("foo", "bar", "traceback") == DBOp("foo", "bar", "traceback")
 
     def test_not_equal_alias(self):
-        assert DBOp("foo", "bar") != DBOp("baz", "bar")
+        assert DBOp("foo", "bar", None) != DBOp("baz", "bar", None)
 
     def test_not_equal_sql(self):
-        assert DBOp("foo", "bar") != DBOp("foo", "baz")
+        assert DBOp("foo", "bar", None) != DBOp("foo", "baz", None)
+
+    def test_not_equal_traceback(self):
+        assert DBOp("foo", "bar", "traceback") != DBOp("foo", "baz", None)
 
 
 class DBRecorderTests(TestCase):
     databases = ("default", "second", "replica")
 
-    def test_default(self):
+    @mock.patch("django_perf_rec.db.traceback.extract_stack", return_value=None)
```

Comment:
It's slightly better, and my preference, to use `mock.patch.object` since that means you have less strings referring to objects to worry about